packages/flutter_tools/lib/src/asset.dart

```diff
@@ -435,6 +485,14 @@ Map<_Asset, List<_Asset>> _parseAssets(
   return result;
 }
 
+_Asset _resolvePackageAsset(
+    String assetBase, String packageName, String asset) {
```

Comment:
style: if there's a line break after the `(`, there should be one before the `)`